catalogue/webapp/services/catalogue/works.js

```diff
@@ -44,10 +45,18 @@ export async function getWorks({
   filters,
   env = 'prod',
 }: GetWorksProps): Promise<CatalogueResultsList | CatalogueApiError> {
-  const filterQueryString = Object.keys(removeEmptyProps(filters)).map(key => {
-    const val = filters[key];
-    return `${key}=${val}`;
-  });
+  const formattedFilters = {
```

Comment:
We should do this in the serialiser as that's why it's there.

Reply:
yeah, except we don't want it to display in the url in the formatted way (I think), as it would be different to the users input

Reply:
Yea, because of the unique situation of having API / URL / Link URLs - we have different serialisers for each. [The API has its own serialisers](https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/services/catalogue/search-params.js#L98), the logic would sit better there.  Maybe moving that code here would make it more clear?